web/static/js/components/user_list_item.jsx

```diff
@@ -8,18 +10,38 @@ import STAGES from "../configs/stages"
 
 const { VOTING } = STAGES
 
-export const UserListItem = ({ user, votes, stage }) => {
+export const UserListItem = ({ user, votes, stage, currentUser, actions }) => {
   const imgSrc = user.picture.replace("sz=50", "sz=200")
   const votesByUser = votes.filter(vote => vote.user_id === user.id).length
   const allVotesIn = votesByUser >= VOTE_LIMIT
+  const renderFacilitatorTransferButton = !user.is_facilitator && currentUser.is_facilitator
 
   let identifier = user.given_name
   if (user.is_facilitator) {
     identifier += " (Facilitator)"
   }
 
+  function passFacilitatorship(user) {
+    if (window.confirm(`Do you want ${identifier} to be Facilitator?`)) {
+      actions.updateRetroAsync({ facilitator_id: user.id })
+    }
+  }
+
   return (
     <li className={`item ${styles.wrapper}`}>
+      {renderFacilitatorTransferButton
+      && (
+        <button
+          type="button"
+          data-tooltip="Transfer facilitatorship"
+          data-inverted=""
+          data-position="bottom right"
+          className={`${styles.facilitator}`}
+          onClick={() => passFacilitatorship(user, currentUser)}
+        >
+          {currentUser.id !== user.id && <i className="ui magic icon" />}
```

Comment:
i still think this might be unnecessary, as the condition above checks whether the current user *is* the facilitator and we only show the button if the user being rendered in the list item *isn't* the facilitator, so their ids would never match. Right? I'll check.

Reply:
double checked, and a couple things came up here. 1) is that we can safely remove the short circuit in `currentUser.id !== user.id` and still get the desired behavior due to the checks described in the above comment, and 2) is that removing the check didn't cause any tests to fail, so make sure you're always test driving to make sure you're testing what you think they are.